scripts/send-comment.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import sys
+import json
+import requests
+
+
+def get_owner_repo(repo_url):
+    pos = repo_url.rfind('/', 0, repo_url.rfind('/'))
+    return repo_url[pos+1:]
+
+def send_message(repo, token, pr_id, msg):
+    url = "https://api.github.com/repos/{}/issues/{}/comments".format(repo, pr_id)
+    payload = {'body': msg}
+    r = requests.post(url,
+                      data=json.dumps(payload),
+                      headers={'Authorization': 'token {}'.format(token)})
+    print("Sending message to PR")
```

Comment:
You would have saved me some trouble if you would have done this instead:

```python
print("Sending message to {}".format(url))
```

hint hint.